thirdparty/assimp/CMakeLists.txt

```diff
@@ -188,6 +188,11 @@ if(desktop)
     # Solve build error using Clang on BSDs
     if(UNIX AND NOT APPLE AND NOT LINUX)
         target_compile_options(${PROJECT_NAME} PRIVATE -fPIC)
+
+        # FBX Importer doesn't build properly on FreeBSD
```

Comment:
This is bad news because of basic embedded geometry stored in this format

Reply:
I’ll be looking further into the StackAllocator linkage issue then. That’s where it fails.

Reply:
Testing out a compiler flag change as suggested here: https://github.com/llvm/llvm-project/issues/81458 . Typically, not using -fPIC should imply -fno-PIC but manually using both it and -fno-pie might solve the issue.

Reply:
Also, if you have no problem with it, I'd like to push an upgrade to Assimp from 5.3.1 to 5.4.1 . Having fixes and updates for file format support would make more sense to keep up to date than most of the other dependencies; there's not much that had to be changed to get it working and some of the new additions might be useful for you (not sure on my end).

Reply:
> Also, if you have no problem with it, I'd like to push an upgrade to Assimp from 5.3.1 to 5.4.1 . Having fixes and updates for file format support would make more sense to keep up to date than most of the other dependencies; there's not much that had to be changed to get it working and some of the new additions might be useful for you (not sure on my end). If there needs to be a change on their end to solve the StackAllocator issue (will still keep looking and testing), having an updated version would also help make it easier to upgrade to a fixed version from them.

Yes no problem

Reply:
I'll push the update after work, then. I found a slight improvement to assimp while testing, so I'll add both together as a separate PR.